src/pages/CheckoutPage.ts

```diff
@@ -0,0 +1,23 @@
+import {Page, Locator, expect} from "@playwright/test"
+
+export class CheckOutPage {
```

Comment:
```suggestion
export class CheckOutPage extends BasePage {
```

And you do not need a constructor here

Reply:
The same for other pages